dle/search/static/search/es_search.js

```diff
@@ -64,12 +69,19 @@
               container: "#hits",
               templates: {
                   item(hit, { html, components }) {
+                      var singleItemUrl = '';
+                      if (globalSearchTerm == '') {
```

Comment:
Existing logic will error if no search term, so we have a check if there's a search term so we don't pass the highlighted search term in scenarios where it shouldn't exist.